server/src/test/java/org/elasticsearch/indices/IndicesRequestCacheIT.java

```diff
@@ -51,7 +51,7 @@
 import static org.hamcrest.Matchers.equalTo;
 import static org.hamcrest.Matchers.greaterThan;
 
-@TestLogging(value = "org.elasticsearch.indices.IndicesRequestCache:TRACE")
+@TestLogging(value = "org.elasticsearch.indices.IndicesRequestCache:TRACE,org.elasticsearch.index.engine.Engine:DEBUG")
```

Comment:
I wonder if we should actually enable trace logging on the Engine for this test? My thought here is with trace logging we might also be able to see if the unexpected flush is waiting for an ongoing flush or if its acquired immediately?